django_urlconfchecks/cli.py

```diff
@@ -0,0 +1,89 @@
+"""CLI module for django_urlconfchecks."""
+import importlib
+import os
+import sys
+from typing import Optional
+
+from django.core.checks import Error
+
+from django_urlconfchecks import __version__
+from django_urlconfchecks.cli_utils import suppress_std
+
+try:
+    import django
+except ImportError:
+    raise ImportError('django_urlconfchecks requires django. Install it with `pip install django`.')
+import typer
+
+app = typer.Typer()
+
+
+def setup_django():
+    """Find the settings.py in current directory and subdirectories.
+
+    Returns:
+        str: the path to the settings.py
+    """
+    if os.getcwd() not in sys.path:
```

Comment:
Actually, I'm not sure about this.
It seems like a dirty hack.
I would appreciate any feedback @spookylukey.